app/javascript/chat/ChatChannelSettings/ChatChannelSettings.jsx

```diff
@@ -305,6 +314,54 @@ export default class ChatChannelSettings extends Component {
     }
   };
 
+  toggelScreens = () => {
```

Comment:
I think this should be `toggleScreens` seems to be a consistent typo throughout the PR.